.github/workflows/plugins-ci.yml

```diff
@@ -120,12 +125,47 @@ jobs:
       - name: Run tests
         run: npm test
 
+  fastify-integration:
+    name: Test Fastify Integration
+    runs-on: ubuntu-latest
+    if: >
+      !failure() &&
+      !cancelled() &&
+      inputs.fastify-integration == true
+    permissions:
+      contents: read
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Install dependencies
+        run: npm i --ignore-scripts
+
+      - name: Test Fastify Integration
+        run: |
+          npm link &&
+          PACKAGE_NAME=$(node -p "require('./package.json').name")
+          git clone https://github.com/fastify/fastify.git --depth 1 &&
+          cd fastify &&
+          npm i --ignore-scripts &&
+          npm link $PACKAGE_NAME &&
+          npm run unit
```

Comment:
not sure about this usage/wording:
if we set the `inputs.fastify-integration=true` on a plugin, nothing will happen here if I'm not wrong.
we will run the fastify's test with a `fastify-postgres` plugin linked

I would just ask chatgpt a clearer var name

Maybe: `fastify-library-integration`?